api/policies/adminSessionAuth.js

```diff
@@ -0,0 +1,30 @@
+const GitHub = require('../services/GitHub');
+const utils = require('../utils');
+
+module.exports = async function adminSessionAuth(req, res, next) {
```

Comment:
Is the idea here that we piggyback off the existing auth, but then do an extra check to see if the user is an admin? Then we put that information into `req.session.authenticatedAt`?

1. Will there be issues with the session between the admin and regular app? Will we actually have to configure the cookie differently? This may depend on whether we go the SPA route which may be hosted on a different subdomain...
2. Can we use something more semantic for admin auth than "authenticatedAt"?

Reply:
This initial PR is keeping to convention with our user auth structure. Currently, I didn't experience any issues between a regular and admin session but still needs to be tested more rigorously. If we break out the passport authentication strategy specifically for admin, this may align better with the SPA route hosted on a different subdomain.

I will test making the `authenticatedAt` more semantic but we may need to add the additional authentication strategy. 🤷‍♂️